src/main/webapp/WEB-INF/views/UserManagement.jsp

```diff
@@ -0,0 +1,122 @@
+<%@ include file="header.jsp" %>
+
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1"%>
```

Comment:
this jsp page declaration should be 1st line on page, also, change default charset in your IDE for jsp to UTF-8